GameData/Kerbalism/System/Signal.cfg

```diff
@@ -50,11 +50,11 @@
 
 @PART[kerbalEVA*]:FOR[Kerbalism]
 {
-	MODULE
-	{
-		name = ModuleCommand
-		minimumCrew = 0
-	}
+	// MODULE
```

Comment:
I'm pretty sure this is needed for the remote control of other vessels, as was intended for the EVA remote control radio so the Kerbal's have the ability to remote control vessels when EVA.
Make sure you test this with CommNet 😉